static/src/components/fieldsconfig/index.vue

```diff
@@ -180,4 +184,14 @@ export default {
   overflow-y: scroll;
   margin-bottom: 20px;
 }
+.tooltip-inner {
```

Comment:
Which tooltips is this for? Should the same style be applied to all tooltips in the website?

Reply:
This is used as "configure instruction", i will delete the style script to keep consistency.